src/cli/commands/start.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * @module cli/commands
+ */
+import fs from "fs";
+import {CliCommand} from "./interface";
+import {CommanderStatic} from "commander";
+import {ILogger, LogLevel, WinstonLogger} from "../../logger";
+import BeaconNode, {BeaconNodeCtx} from "../../node";
+import {RpcClientOverInstance, RpcClientOverWs} from "../../validator/rpc";
+import {ValidatorCtx} from "../../validator/types";
+import {Keypair} from "@chainsafe/bls-js/lib/keypair";
+import Validator from "../../validator";
+import defaults from "../../validator/defaults";
+import {BeaconApi} from "../../rpc/api/beacon";
+import {BeaconDB, ValidatorDB} from "../../db/api";
+import {BeaconChain} from "../../chain";
+import {ValidatorApi} from "../../rpc/api/validator";
+import {PrivateKey} from "@chainsafe/bls-js/lib/privateKey";
+import Keystore from "../../validator/keystore";
+import {promptPassword} from "../../util/io";
+import {BeaconNodeCommand} from "./beacon";
+import {LevelDbController} from "../../db/controller";
+
+interface IStartCommandOptions {
+  db: string;
+  depositContract: string;
+  eth1RpcUrl: string;
+  rpc: string;
+  configFile: string;
+  key: string;
+  dbValidator: string;
+  loggingLevel: string;
+}
+
+export class StartCommand implements CliCommand {
```

Comment:
Oh I may have mislead you, you should add validator logic from here to `beacon.ts`

Reply:
Sorry didn't understand it. What are the validator logic to add?

Reply:
initialization of validator db and rpc should not be in cli command